enterprise-cfengine-guide/federated-reporting.markdown

```diff
@@ -571,55 +571,113 @@ you wish to disable and change the top-level `target_state` property value to `o
 ### Remove Feeder from Mission Portal Hub Management
 
 At this time it is not possible to remove a connected hub in the Mission Portal Hub
-management app. The recommended way to remove a connected hub is with the API.
+management app.
 
-First list all feeders to find the id value. Use of ```jq``` is optional for pretty printing the JSON.
+* List all feeders to find the id value. Use of ```jq``` is optional for pretty printing the JSON.
 
-```console
-$ curl -k -s -X GET -u admin:$PASSWORD https://$SUPERHUB/api/fr/remote-hub | jq '.'
-```
+   (Set approprivate values in your shell for `PASSWORD` and `SUPERHUB`)
+   ```console
+   $ curl -k -s -X GET -u admin:$PASSWORD https://$SUPERHUB/api/fr/remote-hub | jq '.'
```

Comment:
Must you be _admin_? or you must have a role with some specific rbac settings?

Reply:
Certainly you have to have the proper RBAC settings in your role. Admin should take care of that.

Reply:
Added a note near the top. :+1: